package.json

```diff
@@ -104,6 +104,8 @@
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-hooks-testing-library": "^0.5.1",
+    "react-is": "^16.8.6",
```

Comment:
I'm not seeing any usages of `react-is` but I might be dumb?

Reply:
It's a peer dependency for `react-ssr-prepass` ✨